server/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -216,6 +218,62 @@ public void testSimpleCreateTable() {
         assertThat(notNullColumns).hasSize(1);
         assertThat(notNullColumns.get(0)).isEqualTo("name");
     }
+    
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testSimpleCreateTableWithNullConstraint() {
+    	assertThatThrownBy(
+    	() -> analyze("create table foo (id integer primary key null, name string not null)"))
```

Comment:
Please indent this and the following 3 lines by 4 spaces.

Reply:
okay